reference-guide/configuration.adoc

```diff
@@ -1,4 +1,372 @@
 = Configuration Reference
+:icons: image
+:iconsdir: ../images/icons
 include::../partials/attributes.adoc[]
 
-Please see the full https://docs.kogito.kie.org/latest/html_single/#chap-kogito-configuring[configuration reference] at the Kogito Documentation page.
\ No newline at end of file
+== {PRODUCT} Configuration Properties
+
+[.configuration-legend]
+icon:lock[title=Fixed at build time] Configuration property fixed at build time - All other configuration properties are overridable at runtime
+[.configuration-reference.searchable, cols="80,.^10,.^10"]
+|===
+h|[[kie-flyway_configuration]]link:#kie-flyway_configuration[Configuration property]
+h|Type
+h|Default
+
+a| `process`
+[.description]
+--
+String prefix for Kogito CloudEvents type fields.
+Since this is a required field, the constructor will fill them with this default value.
+Ideally, callers would use #TYPE_FORMAT to fill this field using the process name and the signal node name, e.g: process.travelagency.visaapproved
```

Comment:
Hi @pefernan could you check if this is an "application property" ?

Reply:
@gitgabrio @LightGuard  this doesn't look like application property, this comes from the DataEvent metadata.